test/fixtures/rules/std_rule_cases/L030.yml

```diff
@@ -65,3 +65,26 @@ test_fail_func_name_templated_literal_mix:
   # Issue 3022. This was actually a bug in BaseSegment.iter_patches().
   fail_str: SELECT RO(), {{ "t" }}.func()
   fix_str: SELECT RO(), {{ "t" }}.FUNC()
+
+test_pass_ignore_words_regex_simple:
+  pass_str: SELECT MAX(id), f_test_udf(id) FROM TABLE1
+  configs:
+    rules:
+      L030:
+        ignore_words_regex: ^f_
```

Comment:
Since L030 inherits from L010 should the tests be on L010 instead of (or as well as?) L030?

Happy to leave on here as well (since this is the one your really want!), and don't think it's necessary to add tests to ALL the rules adding this config, but would add it to L010 as the base rule for most of these.

And similarly to the other base rules that need this when you do add the, (L029, L057, L059).

Reply:
Yeah, makes a lot of sense. I added new commits addressing this.